egs/wsj/s5/steps/cleanup/create_segments_from_ctm.pl

```diff
@@ -325,7 +325,7 @@ sub ProcessWav {
   # Save the aligned CTM if needed
   if(defined($ACT)){
     for (my $i=0; $i<=$#aligned_ctm; $i++) {
```

Comment:
@chenguoguo or @ngoel17 are the original authors. aligned_ctm might not have been intended to be in CTM format at all. The CTM format, I think, is <wav-id> <channel-id> <start> <end> <word> <confidence>. The new code here does not put the information in that format. <channel-id> is missing and the ordering of the columns is incorrect.